index.md

```diff
@@ -409,7 +405,7 @@ but discussion of those is outside the scope of this paper.
 [noble-rules]: http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1000424
 [repo-url]: https://github.com/swcarpentry/good-enough-practices-in-scientific-computing
 [sandve-reproducible]: http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003285
-[steinmacher-newcomers]: FIXME
+[steinmacher-newcomers]: http://lapessc.ime.usp.br/work.jsf?p1=15673 "Steinmacher et al, The hard life of open source software project newcomers, Procs. 7th Int. Workshop on Cooperative and Human Aspects of Software Engineering. ACM, 2014."
```

Comment:
I think everything after the colon is used as the URL in the link, so the full citation won't work here. Can you just give us the link, and we'll figure out the bibliography when we convert to a proper paper? (I'm still annoyed that Markdown doesn't have a standard format for citations...)